index.html

```diff
@@ -5,8 +5,197 @@
+        </li>
+        <li class="category-item">
+          <a class="category-link" href="#">Умные часы</a>
+        </li>
+        <li class="category-item">
+          <a class="category-link" href="#">Квадрокоптеры</a>
+        </li>
+      </ul>
+    </section>
+
+    <!--Доставка-->
+
+    <section class="delivery">
+      <ul class="menu-delivery">
+        <li>
+          <button class="button-delivery" type="button">Доставка</button>
+        </li>
+        <li>
+          <button class="button-warranty" type="button">Гарантия</button>
+        </li>
+        <li>
+          <button class="button-credit" type="button">Кредит</button>
+        </li>
+      </ul>
+      <h2>Доставка</h2>
+      <p>Мы с удовольствием доставим ваш товар прямо
+        к вашему подъезду совершенно бесплатно! Ведь
+        мы неплохо заработаем, поднимая его на ваш этаж.</p>
+    </section>
+
+    <div class="order">
+      <a class="order-link" href="#">Доставим редкий товар под заказ</a>
+    </div>
+
+    <!--Контакты-->
+    <section class="info">
+      <div class="about-us">
+        <h2>О нас</h2>
+        <p>Огромный выбор гаджетов не оставит равнодушным
+          гика, который есть в каждом из нас.
+        </p>
+        <p>Мы можем доставить ваш товар в самые отдалённые
+           уголки России транспортными компаниями:
+        </p>
+        <ul class="delivery-services">
+          <li class="delivery-item">Меловые линии</li>
+          <li class="delivery-item">Сдик</li>
+          <li class="delivery-item">ТоксБерри</li>
+        </ul>
+        <a class="more-about-us" href="#">Подробнее о нас</a>
+      </div>
+      <div class="contacts">
+        <h2 class="contacts-title">Контакты</h2>
+        <p class="contacts-text">Вы можете забрать товар сами, заехав в наш офис.
+          Заодно сможете проверить работоспособность покупки.
+        </p>
+        <h3 class="contacts-subtitle">Интернет-магазин «Девайс»</h3>
+        <p class="contacts-address-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+        <a class="contacts-phone" href="tel:+7(812)812-12-12">+7 (812) 5812-12-12</a>
+        <h3 class="contacts-subtitle">Время работы</h3>
+        <ul class="contacts-schedule">
+          <li class="contacts-schedule-item">пн—пт: с 10:00 до 22:00</li>
+          <li class="contacts-schedule-item">сб—вс: с 10:00 до 19:00</li>
+        </ul>
+        <a class="write-to-us">Напишите нам</a>
+      </div>
+    </section>
+
+    <!--Подписка-->
+    <section class="subscription">
+      <h2 class="subscription-title">Подпишитесь на рассылку</h2>
+      <p class="subscription-text">Еженедельный дайджест с обзорами новинок,
+        длительными тест-драйвами, распаковками, а также акции,
+        спецпредложения и вот это вот всё.</p>
+        <form class="subscription-form" action="https://echo.htmlacademy.ru">
+          <input class="subscription-string" type="e-mail" placeholder="Ваш e-mail">
```

Comment:
Не `e-mail`, а `email`.
```suggestion
          <input class="subscription-string" type="email" placeholder="Ваш e-mail">
```